src/components/Posts/AllPost/AllPost.js

```diff
@@ -0,0 +1,158 @@
+            {
+              <img
+                style={{ display: 'none' }}
+                src="https://source.unsplash.com/user/erondu"
+                alt="background"
+              />
+            }
+            <div className={classes.overlay} />
+            <Grid container>
+              <Grid item md={6}>
+                <div className={classes.mainFeaturedPostContent}>
+                  <Chip label={main && main.label} size="small" color="primary" />
+                  <Typography component="h1" variant="h3" color="inherit" gutterBottom>
+                    {main && main.title}
+                  </Typography>
+                  <Typography variant="h5" color="inherit" paragraph>
+                    {main && `${main.content.substring(0, 100)}`}
+                  </Typography>
+                  <Typography variant="subtitle1" color="inherit" gutterBottom>
+                    {main && `${main.updatedAt
+                      ? main.updatedAt.substr(0, main.updatedAt.indexOf('T'))
+                      : main.createdAt.substr(0, main.createdAt.indexOf('T'))} by ${main && main.username}`}
+                  </Typography>
+                  <Button
+                    variant="contained"
+                    color="primary"
+                    component={Link}
+                    to={main && `/viewpost/${main.postID}`}
+                  >
+                    Continue Reading...
+                  </Button>
+                </div>
+              </Grid>
+            </Grid>
+          </Paper>
+          <Grid container spacing={4}>
+            {subPosts.map((post) => (
+              <Grid item key={post.title + post.createdAt} xs={12} md={6}>
+                <CardActionArea component={Link} to={`/viewpost/${post.postID}`}>
+                  <Card className={classes.card}>
+                    <div className={classes.cardDetails}>
+                      <CardContent>
+                        <Chip label={post.label} size="small" color="primary" />
+                        <Typography component="h2" variant="h5">
+                          {post.title}
+                        </Typography>
+                        <Typography variant="subtitle1" paragraph>
+                          {`${post.content.substring(0, 100)}...`}
+                        </Typography>
+                        <Typography variant="subtitle1" color="textSecondary">
+                          {`${post.updatedAt
+                            ? post.updatedAt.substr(0, post.updatedAt.indexOf('T'))
+                            : post.createdAt.substr(0, post.createdAt.indexOf('T'))} by ${post.username}`}
+                        </Typography>
+                      </CardContent>
+                    </div>
+                    <Hidden xsDown>
+                      <CardMedia
+                        className={classes.cardMedia}
+                        image="https://source.unsplash.com/random"
```

Comment:
the same as above, but up to you. I do have one experience that, the online image is disappear and I need to find another one right before the presentation